src/Conversion/ONNXToKrnl/NN/Conv.cpp

```diff
@@ -30,6 +31,7 @@ struct ONNXConvOpLowering : public ConversionPattern {
       MemRefType &memRefType, Value alloc) const {
     auto loc = convOp.getLoc();
     KrnlBuilder createKrnl(rewriter, loc);
+    SCFBuilder createScf(rewriter, loc);
```

Comment:
There are quite a few builder. The preferred way is to do a 
```C++
MultiDialectBuilder<KrnlBuilder, MathBuilder, MemRefBuilder> create(rewriter, loc);
```
where you put the types you need. See Dialect/MLIR/builder.hpp or in the code for examples